sites/staging-api.snapcraft.io.yaml

```diff
@@ -53,5 +53,6 @@ nginxConfigurationSnippet: |
   more_set_headers "Link: <https://assets.ubuntu.com>; rel=preconnect; crossorigin, <https://assets.ubuntu.com>; rel=preconnect";
```

Comment:
Missing `production` replicas here aren't we?

Reply:
``` bash
 $ ./konf.py production sites/staging-api.snapcraft.io.yaml 
Traceback (most recent call last):
  File "./konf.py", line 157, in <module>
    print(projectConfig.render())
  File "./konf.py", line 121, in render
    deployment_env=self.deployment_env,
  File "/home/robin/Projects/deployment-configs/env3/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/home/robin/Projects/deployment-configs/env3/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/robin/Projects/deployment-configs/env3/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/robin/Projects/deployment-configs/env3/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "./templates/site.yaml", line 15, in top-level template code
    {% include 'deployment.yaml' %}
  File "./templates/deployment.yaml", line 21, in top-level template code
    containerPort: {{ data.containerPort | default(80) }}
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'replicas'
```

Reply:
But this project only exits in the staging namespace. We don't need to generate the production output, right?

Reply:
:+1: